### DISCUSSION TITLE
Monika isn't able to read my gifts and I don't know why.

### DISCUSSION THREAD

**Discussion by TheUnknownzzx** [Monika-After-Story/MonikaModDev#9171] [QA - tech support + general]
I have followed every single video I see on YouTube on why it isn't working, and I still cant get it to work. I use an MacBook Air but I am hoping that has nothing to do with this situation.

**Comment by Wingdinggaster656** [comment]
Are you sure that you have the correct extension  name?
Also Monika will not react to the gift right away. It takes about 1 min.

**Reply by Wingdinggaster656** [reply]
Oh, wait, you are on Mac?
I don't know much about Mac. But it seems like gifts are windows-only.
